rest/model/conversions.go

```diff
@@ -0,0 +1,84 @@
+package model
+
+import (
+	"go/types"
+
+	"github.com/pkg/errors"
+)
+
+type convertType string
+
+func stringToString(in string) string {
+	return in
+}
+
+func stringToStringPtr(in string) *string {
+	return &in
+}
+
+func stringPtrToString(in *string) string {
+	if in == nil {
+		return ""
+	}
+	return *in
+}
+
+func stringPtrToStringPtr(in *string) *string {
+	return in
+}
+
+func intToInt(in int) int {
+	return in
+}
+
+func intToIntPtr(in int) *int {
+	return &in
+}
+
+func intPtrToInt(in *int) int {
+	if in == nil {
+		return 0
+	}
+	return *in
+}
+
+func intPtrToIntPtr(in int) *int {
```

Comment:
This function should take `in *int` and return in, I believe

Reply:
oops fixed